lib/ReactViews/PrivateIndicator/PrivateIndicator.jsx

```diff
@@ -19,13 +19,19 @@ export default function PrivateIndicator(props) {
       inWorkbench={props.inWorkbench}
       css={`
         margin-top: -1px;
-        ${(p) => p.inWorkbench && `margin-right: 2px;`}
+        ${(p) =>
+          p.inWorkbench &&
+          `
+          position: absolute;
+          margin-right: 2px;
+          right: 20px;
+        `}
 
         svg {
           width: 15px;
           height: 15px;
           fill: ${(p) =>
-            p.inWorkbench ? p.theme.textLight : p.theme.colorPrimary};
+            p.inWorkbench ? p.theme.textLight : p.theme.charcoalGrey};
         }
```

Comment:
Thanks Mike. 

It looks good for most workbench items - but for some the icon overlaps with text.
There is a Flex box for the workbench title and arrow icon - is it possible to put the `PrivateIndicator` into the flex box?

<img width="369" alt="image" src="https://github.com/TerriaJS/terriajs/assets/6187649/84b1ace7-d071-4e8c-a754-8d8255b73a85">

Reply:
Will look into it. Any tip on the usage of Flex box?